evap/contributor/forms.py

```diff
@@ -98,25 +101,22 @@ def __init__(self, *args, **kwargs):
         # fix generated form
         self.fields['delegates'].required = False
         self.fields['delegates'].help_text = ""
+        self.fields['cc_users'].widget.attrs['disabled'] = True
+        self.fields['cc_users'].help_text = ""
 
-    def clean_email(self):
-        email = self.cleaned_data.get('email')
-        user_with_same_email = UserProfile.objects.filter(email__iexact=email)
-
-        # make sure we don't take the instance itself into account
-        if self.instance and self.instance.pk:
-            user_with_same_email = user_with_same_email.exclude(pk=self.instance.pk)
-
-        if user_with_same_email.exists():
-            raise forms.ValidationError(_("A user with the email '%s' already exists") % email)
-        return email.strip().lower()
-
-    def clean_first_name(self):
-        return self.cleaned_data['first_name'].strip()
+        represented_users = self.instance.represented_users.all()
+        self.fields['delegate_of'] = forms.ModelMultipleChoiceField(represented_users, initial=represented_users, help_text="", required=False)
+        self.fields['delegate_of'].widget.attrs['disabled'] = True
 
-    def clean_last_name(self):
-        return self.cleaned_data['last_name'].strip()
+        ccing_users = self.instance.ccing_users.all()
+        self.fields['cc_user_of'] = forms.ModelMultipleChoiceField(ccing_users, initial=ccing_users, help_text="", required=False)
+        self.fields['cc_user_of'].widget.attrs['disabled'] = True
 
-    def clean_title(self):
-        return self.cleaned_data['title'].strip()
+    def clean_cc_users(self):
+        return self.instance.cc_users.all()
```

Comment:
the others are not part of the model and are filled manually. they are not saved on saving the form.